README.md

```diff
@@ -95,3 +95,68 @@ You can also specify multiple namespaces when using the hook:
 ```
   const t = useTranslation(['common', 'distribution']);
 ```
+
+## Android App
+
+### Build the remote server lib
+
+The Android app needs the remote server build as a shared lib.
+
+Make sure the Android NDK is installed and the env var `NDK_HOME` is set, for example to `~Android/Sdk/ndk/22.1.7171670`.
+Currently the build script requires the Android API 26, make sure this version is installed.
+
+The remote server source code should be located in the `../remote-server` directory.
+If not clone it by doing:
+
+```
+cd ..
+git clone git@github.com:openmsupply/remote-server.git
+```
+
+Unsolved issue that will hopefully be fixed at some point:
+
+1. in `../remote-server/Cargo.toml` add the "vendored" feature to openssl by changing the line:
```

Comment:
```suggestion
1. in `../remote-server/server/Cargo.toml` add the "vendored" feature to openssl by changing the line:
```

Reply:
something went wrong in the last merge, this is not needed anymore; the feature is added automatically in remote-server when compiling the android target